reader_concurrency_semaphore_group.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2021-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include "reader_concurrency_semaphore_group.hh"
+
+// Calling adjust is serialized since 2 adjustments can't happen simultaneosly,
```

Comment:
could you please take this opportunity to fix the typo?

Reply:
No, please do not fix typos. It will cause me pain when I merge back into enterprise.

The goal of the migrations is to make master == enterprise, not to fix typos, bugs, misfeatures, or anything.